packages/gatsby-transformer-documentationjs/src/gatsby-node.js

```diff
@@ -17,49 +11,35 @@ const stringifyMarkdownAST = (node = ``) => {
   }
 }
 
-const commentId = (parentId, commentNumber) =>
-  `documentationJS ${parentId} comment #${commentNumber}`
+const docId = (parentId, docsJson) =>
+  `documentationJS ${parentId} path #${JSON.stringify(docsJson.path)}`
 const descriptionId = (parentId, name) =>
   `${parentId}--DocumentationJSComponentDescription--${name}`
 
-function createDescriptionNode(
-  node,
-  docNodeId,
-  markdownStr,
-  name,
-  actions,
-  createNodeId
-) {
-  const { createNode } = actions
+function prepareDescriptionNode(node, markdownStr, name, helpers) {
```

Comment:
Because I don't actually use `createNode` here, the name change is to signify that `createNode` is not called here without having to look into function itself. It's something I picked up from `gatsby-source-contentful` and I think that this naming convention does make a lot of sense